handlers/publicationcreating/import.go

```diff
@@ -75,27 +77,29 @@ type YieldValidationErrors struct {
 	Errors form.Errors
 }
 
-func (h *Handler) Add(w http.ResponseWriter, r *http.Request, ctx Context) {
-	tmpl := ""
+func Add(w http.ResponseWriter, r *http.Request, legacyContext Context) {
+	c := ctx.Get(r)
+
+	const pageTitle = "Add - Publications - Biblio"
```

Comment:
Do we need to pass on the title from the handler or can we do that from the concrete template (add_manual.templ, add_wos.templ, add_bibtex.templ,...)

Reply:
@verheyenkoen I just put them in the template